packages/adblocker/src/filters/network.ts

```diff
@@ -1286,6 +1298,7 @@ export default class NetworkFilter implements IFilter {
     optionValue,
     rawLine,
     regex,
+    isCaseSensitive = false,
```

Comment:
From constructor, it must be `false` by default. It's questionable if we need to rename it to `isRegexCaseSensitive` or something that implies it's only for "full regexp filters".